kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-01T00:00:00Z
+---
+
+`DeadlineExceeded` is a Context error that originates in `gRPC`.
+This error occurs when a request hasn't been replied to before its timeout.
+
```

Comment:
```
context: deadline exceeded
```

Maybe grab a real instance of this?

If you spin up docker compose and kill elasticsearch you will get it I am pretty sure.

Reply:
Aarohi and I are grabbing some examples across the board